participants/ilayni.json

```diff
@@ -2,11 +2,16 @@
     "name": "Ilay nisim",
     "linkedin": "ilay-nisim-9561369",
     "github": "ilayni",
+    "github_page": true,
     "posts": [
         {
             "title": "Coolcode",
             "url": "https://cool8200.wordpress.com/2023/01/10/open-source/",
             "published_at": "2023-01-08T20:29:48Z"
         }
+      ],
+    "projects": [
+      "https://github.com/realpython",
+      "https://github.com/realpython/materials"
```

Comment:
I will accept this, though I don't think there is a point including the repo link if you already include the link to the whole organization.